styleguide/components/31-molecules/contact-block/contact-block.config.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+  title: 'Contact block',
+  default: 'contact-block-with-phone',
+  status: 'alpha',
+  handle: 'contact-block',
+  collated: true,
+  collator: function (markup, item) {
+    return `<!-- Start: @${item.handle} -->\n<dt>${item.name}</dt><dd>${markup}</dd>\n<!-- End: @${item.handle} -->\n`;
+  },
+  context: {
+    title: 'Contact us',
+    description: 'Nulla nec eros lorem. Morbi at augue eu purus congue auctor. Sed id volutpat odio. Nunc congue diam vel diam venenatis, ut consequat urna dapibus. Vivamus at lobortis odio. Sed felis mauris, mattis eget purus at, pharetra ullamcorper risus.',
+    tel: '09 123 45 67',
+    email: null,
+    link: '#'
+  },
+  variants: [
+    {
+      name: 'contact-block-with-phone'
```

Comment:
Missing preview property

Reply:
See above comment.

Reply:
Variants will default to the molecule preview template (should be description list, since it's a collation).
But the variants 'an sich' should use the regular preview.

Reply:
Fixed.